samples/bluetooth/mtu_update/central/src/central_mtu_update.c

```diff
@@ -205,7 +205,12 @@ static void connected(struct bt_conn *conn, uint8_t err)
 
 	printk("Connected: %s\n", addr);
 
-	(void)mtu_exchange(conn);
+	struct bt_conn_info info = {0};
```

Comment:
Suggest to declare variables at top of scope (i.e. move this next to `addr` in line 192)